types/express-serve-static-core/.eslintrc.json

```diff
@@ -2,6 +2,7 @@
     "rules": {
         "@definitelytyped/no-type-only-packages": "off",
         "@typescript-eslint/naming-convention": "off",
-        "@typescript-eslint/no-empty-interface": "off"
+        "@typescript-eslint/no-empty-interface": "off",
+        "@typescript-eslint/no-misused-promises": "error"
```

Comment:
Pls do not add any more rules to eslint. The good thing to do here would be to remove all the overrides here and instead fix the errors if any,

Reply:
I have reverted my changes to this file in f9f4929abf so that this pull request no longer tests its own change, as per your request.

Fixing the overrides would be out of scope for this pull request.